packages/ui-library/src/components/forms/textarea/index.ts

```diff
@@ -187,18 +194,22 @@ ${this.value}
           </textarea
           >
           <div class="${textareaInfoContainer}">
-            ${this.hasHint || this.hasError
-              ? BlrFormCaptionGroupRenderFunction({ size: this.size }, captionContent)
-              : nothing}
-            ${this.showCounter
-              ? BlrCounterRenderFunction({
-                  variant: counterVariant,
-                  current: this.count,
-                  max: this.maxLength || 0,
-                  size: this.size,
-                  theme: this.theme,
-                })
-              : nothing}
+            ${
+              this.hasHint || this.hasError
+                ? BlrFormCaptionGroupRenderFunction({ size: this.size }, captionContent)
+                : nothing
+            }
+            ${
+              this.showCounter
+                ? BlrCounterRenderFunction({
```

Comment:
In other places for the renderFunction the 'html' is used, example in this file packages/ui-library/src/components/forms/textarea/index.ts in line 159 but here for this renderFunction 'html' thingy is not there.  I think if 'html' is not required then it's better to be removed.

Reply:
Ok, thanks for the clarification.